src/tiffIO.cpp

```diff
@@ -160,12 +160,21 @@ tiffIO::tiffIO(char *fname, DATA_TYPE newtype) {
 	if (datatype == SHORT_TYPE) {
```

Comment:
Flowdircond should not be changing tiffIOI.cpp.  It should be using what is the latest from develop branch.